components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -2731,7 +2722,12 @@ def groupSummary(self, gid=None, exclude_self=False):
         :return:    {'leaders': list :class:`ExperimenterWrapper`,
                      'colleagues': list :class:`ExperimenterWrapper`}
         :rtype:     dict
```

Comment:
Could you use the new groupSummary method from here, to avoid the code duplication etc.
```
return default.groupSummary(exclude_self)
```
Was also wondering if/where we use 'exclude_self = True'. It was there when I originally added this, but I don't remember ever using it.